homework9/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import NavItem from "../NavItem";
+
+const Footer = () => {
+  return (
+    <footer>
+      <div className="footer--container">
+        <div>
+          <p>©Simple Art, Inc. 2021</p>
```

Comment:
can u change 2021 for current year using JS

Reply:
Ok thanks